extensions/settings/pkg/config/config.go

```diff
@@ -39,7 +39,7 @@ type Metadata struct {
 	GatewayAddress string `yaml:"gateway_addr" env:"STORAGE_GATEWAY_GRPC_ADDR"`
 	StorageAddress string `yaml:"storage_addr" env:"STORAGE_GRPC_ADDR"`
 
-	ServiceUserID     string `yaml:"service_user_id" env:"METADATA_SERVICE_USER_UUID"`
-	ServiceUserIDP    string `yaml:"service_user_idp" env:"METADATA_SERVICE_USER_IDP"`
+	SystemUserID      string `yaml:"system_user_id" env:"OCIS_SYSTEM_USER_ID;METADATA_SYSTEM_USER_UUID"`
+	SystemUserIDP     string `yaml:"system_user_idp" env:"OCIS_URL;OCIS_SYSTEM_USER_IDP;METADATA_SYSTEM_USER_IDP"`
```

Comment:
we should not use OCIS_URL here as we default to `internal`. If someone sets `OCIS_URL` it would break the config